.github/workflows/ci_meetings_system_public.yml

```diff
@@ -25,6 +25,7 @@ env:
   RUBY_VERSION: 2.7.1
   NODE_VERSION: 16.9.1
   DECIDIM_MODULE: decidim-meetings
+  BIG_SCREEN_SIZE: "true"
```

Comment:
Nice!  👏🏽 👏🏽 

Last week, I was trying to solve this flaky `decidim-meetings` spec locally. This seems like a quick mitigation, I didn't know about the `BIG_SCREEN_SIZE` env var